kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/PortForwardExample.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.fabric8.kubernetes.examples;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.ConfigBuilder;
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.LocalPortForward;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetSocketAddress;
+import java.nio.channels.SocketChannel;
+
+public class PortForwardExample {
+  private static final Logger logger = LoggerFactory.getLogger(PortForwardExample.class);
+
+  public static void main(String args[]) {
+    String master = "https://localhost:8443";
+
+    Config config = new ConfigBuilder().withMasterUrl(master).build();
+    try (final KubernetesClient client = new DefaultKubernetesClient(config)) {
+      String namespace = "default";
+      log("namespace", namespace);
+      Pod pod = client.pods().inNamespace(namespace).load(PortForwardExample.class.getResourceAsStream("/portforward-example-pod.yml")).get();
+      client.pods().inNamespace(namespace).create(pod);
+      log("Pod created");
+
+      int containerPort =  pod.getSpec().getContainers().get(0).getPorts().get(0).getContainerPort();
+
+      LocalPortForward portForward = client.pods().withName("testpod").portForward(containerPort, 8080);
+      log("Port forwarded");
+
+      int localPort = portForward.getLocalPort();
+      SocketChannel channel = SocketChannel.open();
+      channel.connect(new InetSocketAddress("localhost", localPort));
```

Comment:
Were you able to run this? I tried but facing this error:
```
13:23:05.999 [io.fabric8.kubernetes.examples.PortForwardExample.main()] INFO  io.fabric8.kubernetes.examples.PortForwardExample - Port forwarded
13:23:06.001 [io.fabric8.kubernetes.examples.PortForwardExample.main()] INFO  io.fabric8.kubernetes.examples.PortForwardExample - localhost/127.0.0.1:8080
13:23:06.008 [pool-2-thread-1] ERROR io.fabric8.kubernetes.client.dsl.internal.PortForwarderWebsocket - FWD: Throwable received from websocket
java.io.InterruptedIOException: executor rejected
	at okhttp3.RealCall$AsyncCall.executeOn(RealCall.java:185) ~[okhttp-3.12.0.jar:?]
	at okhttp3.Dispatcher.promoteAndExecute(Dispatcher.java:186) ~[okhttp-3.12.0.jar:?]
	at okhttp3.Dispatcher.enqueue(Dispatcher.java:137) ~[okhttp-3.12.0.jar:?]
	at okhttp3.RealCall.enqueue(RealCall.java:126) ~[okhttp-3.12.0.jar:?]
	at okhttp3.internal.ws.RealWebSocket.connect(RealWebSocket.java:193) ~[okhttp-3.12.0.jar:?]
	at okhttp3.OkHttpClient.newWebSocket(OkHttpClient.java:435) ~[okhttp-3.12.0.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.PortForwarderWebsocket.forward(PortForwarderWebsocket.java:191) ~[kubernetes-client-4.3-SNAPSHOT.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.PortForwarderWebsocket.lambda$forward$0(PortForwarderWebsocket.java:162) ~[kubernetes-client-4.3-SNAPSHOT.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.util.concurrent.RejectedExecutionException: Task okhttp3.RealCall$AsyncCall@b285a1d rejected from java.util.concurrent.ThreadPoolExecutor@7d4395fe[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]
	at java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2055) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:825) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.execute(ThreadPoolExecutor.java:1355) ~[?:?]
	at okhttp3.Rea
```

Reply:
Yes, it is working for me

Reply:
Could you please tell me steps which you followed?

Reply:
I ran it on this project: 
https://github.com/kunal-kushwaha/client-demo/blob/master/src/main/java/PortForwardExample.java

Reply:
Umm, I don't see any significant difference between these two files. What's different here?

Reply:
There's no difference, I ran the same file.
I just included fabric dependencies in this demo project that I used to test examples.
And ran the same file I pushed on the repo

Reply:
@rohanKanojia 
Is the issue resolved? How do I reproduce this error?

Reply:
ah, I didn't get a chance to jump on this again. Will reproduce it soon and report back

Reply:
Try running file using exec-maven-plugin

Reply:
@rohanKanojia: Are you still facing this error? I was unable to reproduce it.

Reply:
I will take a look again here.